### DISCUSSION TITLE
Unable to see traces in Jaeger-UI

### DISCUSSION THREAD

**Discussion by charan906** [open-telemetry/opentelemetry-collector#10364] [General]
Hello Team,

I am using opentelemetry collector(v1.99.0) , when I generated spans and stored in elastic search backend using Otel-collector , I can see spans are stored in backend. When i am checking in Jaeger UI I am not seeing spans data . I am using otlphttp exporter to store traces in backend.

Is there any format needed by Jaeger UI to fetch span data ??
why open telemetry span data is not visualized in Jaeger-UI ??
What are the steps need to follow to ensure to visualize  Otel data in Jaeger UI ??

**Comment by yurishkuro** [comment]
Jaeger only works with spans stored by Jaeger. From your description it sounds like you exported data to ES directly from OTEL-collector, probably using es exporter from contrib, which is not compatible with Jaeger.

**Comment by charan906** [comment]
Hello @yurishkuro ,Thanks for the reply,
 Is there any way that we can make OTEL-collector data in compatible with Jaeger UI ?? , Is it possible .

**Reply by yurishkuro** [reply]
You can configure OTEL collector to use OTLP exporter and send data to Jaeger collector, which will save it in the backend in Jaeger format. If you're asking about making Jaeger compatible with data format used by some 3rd party exporter you use in OTEL collector, then we have no plans to support such setup. However, you can implement your own adapter (gRPC service) exposing Jaeger Remote Storage API that can support any custom data formats & custom backends.

**Comment by charan906** [comment]
Thanks @yurishkuro  for the reply.
